torch/utils/_runtime_estimation.py

```diff
@@ -122,9 +114,16 @@ def get_num_bytes(t: torch.Tensor) -> int:
     Returns:
         int: The memory consumption of the tensor in bytes.
     """
-    num_bytes = t.untyped_storage().nbytes()
-    mem_consumed = math.ceil(num_bytes / _PYTORCH_MIN_ALLOCATE) * _PYTORCH_MIN_ALLOCATE
-    return mem_consumed
+    real_numel = 1
+    for size, stride in zip(t.shape, t.stride()):
```

Comment:
Please pass strict=True to zip

Reply:
if shape and stride have different lengths we're beyond saving haha